Microsoft.Azure.CosmosRepository/src/IRepository.cs

```diff
@@ -61,6 +62,17 @@ public interface IRepository<TItem> where TItem : IItem
             PartitionKey partitionKey,
             CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Gets an <see cref="PagedResult{TItem}"/> of <see cref="IItem"/>
+        /// </summary>
+        /// <param name="predicate">The expression used for evaluating a matching item.</param>
+        /// <param name="pagedResultRequest">The paging object.</param>
+        /// <param name="cancellationToken">The cancellation token to use when making asynchronous operations.</param>
+        /// <returns>A paged result of item instances who meet the <paramref name="predicate"/> and <paramref name="pagedResultRequest"/> condition.</returns>
+        Task<PagedResult<TItem>> GetListAsync<TPagedRequest>(
+            Expression<Func<TItem, bool>> predicate, TPagedRequest pagedResultRequest,
+            CancellationToken cancellationToken = default);

```

Comment:
```suggestion
        Task<PagedResult<TItem>> GetListAsync<TPagedRequest>(
            Expression<Func<TItem, bool>> predicate,
            TPagedRequest pagedResultRequest,
            CancellationToken cancellationToken = default);
```

Apply consistent wrapping of parameters.